extension/src/experiments/model/index.ts

```diff
@@ -39,10 +40,11 @@ export type SelectedExperimentWithColor = Experiment & {
 }
 
 export enum ExperimentType {
-  WORKSPACE = 'workspace',
   COMMIT = 'commit',
   EXPERIMENT = 'experiment',
-  QUEUED = 'queued'
+  RUNNING = 'running',
```

Comment:
[F] We need this to prevent actions being run from the `experimentsTree` context menu for running experiments.